manifest.json

```diff
@@ -19,36 +19,24 @@
         "mysql"
     ],
     "requirements": {
-      "yunohost": ">= 3.8.1"
+      "yunohost": ">= 4.1.7"
```

Comment:
@ericgaspar I missed that the PR fixing linter warnings also bumps the version requirement to Yunohost 4.0.

As updating Yunohost requires a whole Debian upgrade, I'd prefer to stay compatible with Stretch / Yunohost 3.* for a little longer.

It is fine with you if I:

1. revert the PR that introduced this (so that we can get this version of Mattermost merged),
2. we discuss after that of an upgrade calendar and strategy?

Reply:
Absolutely. You can merge #239  if it's OK with you.